guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/binary-agents/index.md

```diff
@@ -147,6 +147,25 @@ Make sure that each time you transcode a character from binary to decimal, you r
 
 *   <a href='http://forum.freecodecamp.com/t/javascript-array-prototype-map/14294' target='_blank' rel='nofollow'>Array.prototype.map</a>
 
+## ![:rotating_light:](https://forum.freecodecamp.com/images/emoji/emoji_one/rotating_light.png?v=3 ":rotating_light:") Alternative Advanced Code Solution:
+```javascript
+       const binaryAgent = str => str.replace(/\d+./g, char => String.fromCharCode(`0b${char}`));
+       
+       binaryAgent("01000001 01110010 01100101 01101110 00100111 01110100 00100000 01100010 01101111 01101110 01100110 01101001 01110010 01100101 01110011 00100000 01100110 01110101 01101110 00100001 00111111");
+```
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/CLnp/0' target='_blank' rel='nofollow'>Run Code</a>
```

Comment:
```suggestion
```

Reply:
We're no longer accepting PRs adding links which are not fCC controlled

Reply:
Thank you for the information, I had just copy/pasted that. I won't include it in the future.